### PR TITLE
Serverspec gen improve and Update dependencies

### PR DESCRIPTION
**Pull request by pocke**
Goal
-----

- [x] infra id 
  - [x] Add infra id when save
  - [x] redirect to serverspec#index with infra id after save spec
  - [x] redirect to serverspec#index with infra id when cancel 
  - [x] Title with infra name
- [x] Update dependencies
  - [x] bundle update
  - [x] package.json fix (devDependencies/dependencies)
  - [x] update gulp-typescript
  - [x] ajax_set revision is fixed

Update process
---------

```sh
bundle update
bower i
cd frontend/
npm i
cd ..
```

### CODE REVIEW

**Comment by pocke** [comment]
@Joeper214 please review and merge it :bow:

**Comment by Joeper214** [comment]
:+1:

File: bower.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "favico.js": "~0.3.7",
     "bootstrap": "~3.3.4",
     "bootswatch-dist": "paper",
-    "ajax_set.js": "https://github.com/pocke/ajax_set.js.git",
+    "ajax_set.js": "https://github.com/pocke/ajax_set.js.git#106d927aba64fa99220a1199973d41f2a448bd2c",
```
Comment (Joeper214) [inline bower.json]: :metal:

**Comment by Joeper214** [comment]
LGTM!   :+1:

**Pull request merged**